src/tests/tribol_proximity_check.cpp

```diff
@@ -0,0 +1,274 @@
+    SetUpCommonPlaneProblem( std::get<0>( mesh ), std::get<1>( mesh ), std::get<2>( mesh ), penalty, binning_proximity,
+                             binning_method );
+  }
+
+  void SetUpMortarProblem( tribol::BinningMethod binning_method )
+  {
+    // binning proximity value
+    auto binning_proximity = std::get<1>( GetParam() );
+
+    auto mesh = SetUp3DMesh();
+
+    // set up tribol
+    constexpr int coupling_scheme_id = 0;
+    constexpr int mesh1_id = 0;
+    constexpr int mesh2_id = 1;
+    tribol::registerMfemCouplingScheme( coupling_scheme_id, mesh1_id, mesh2_id, std::get<0>( mesh ),
+                                        std::get<0>( mesh ).getNodes(), std::get<1>( mesh ), std::get<2>( mesh ),
+                                        tribol::SURFACE_TO_SURFACE, tribol::NO_CASE, tribol::SINGLE_MORTAR,
+                                        tribol::FRICTIONLESS, tribol::LAGRANGE_MULTIPLIER, binning_method );
+    tribol::setLagrangeMultiplierOptions( coupling_scheme_id, tribol::ImplicitEvalMode::MORTAR_RESIDUAL );
+    tribol::getMfemPressure( coupling_scheme_id ) = 1.0;
+    tribol::setBinningProximityScale( coupling_scheme_id, binning_proximity );
+
+    UpdateTribol( std::get<0>( mesh ), coupling_scheme_id );
+  }
+};
+
+TEST_P( ProximityTest, CheckForceValues2DCommonPlane )
+{
+  auto should_have_force = std::get<3>( GetParam() );
+
+  for ( auto binning_method : binning_methods_ ) {
+    std::cout << "Binning method = " << binning_method << std::endl;
+    SetUp2DCommonPlaneProblem( binning_method );
+    std::cout << "  max_force_ = " << max_force_ << std::endl;
+    if ( should_have_force ) {
+      EXPECT_GT( max_force_, 1.0e-6 );
+    } else {
+      EXPECT_LT( max_force_, 1.0e-15 );
+    }
+  }
+
+  MPI_Barrier( MPI_COMM_WORLD );
+}
+
+TEST_P( ProximityTest, CheckForceValues3DCommonPlane )
+{
+  auto should_have_force = std::get<3>( GetParam() );
+
+  for ( auto binning_method : binning_methods_ ) {
+    std::cout << "Binning method = " << binning_method << std::endl;
+    SetUp3DCommonPlaneProblem( binning_method );
+    std::cout << "  max_force_ = " << max_force_ << std::endl;
+    if ( should_have_force ) {
+      EXPECT_GT( max_force_, 1.0e-6 );
+    } else {
+      EXPECT_LT( max_force_, 1.0e-15 );
+    }
+  }
+
+  MPI_Barrier( MPI_COMM_WORLD );
+}
+
+TEST_P( ProximityTest, CheckForceValues3DMortar )
+{
+  auto should_have_force = std::get<3>( GetParam() );
+
+  for ( auto binning_method : binning_methods_ ) {
+    std::cout << "Binning method = " << binning_method << std::endl;
+    SetUpMortarProblem( binning_method );
+    std::cout << "  max_force_ = " << max_force_ << std::endl;
+    if ( should_have_force ) {
+      EXPECT_GT( max_force_, 1.0e-6 );
+    } else {
+      EXPECT_LT( max_force_, 1.0e-15 );
+    }
+  }
+
+  MPI_Barrier( MPI_COMM_WORLD );
+}
+
+INSTANTIATE_TEST_SUITE_P(
+    tribol, ProximityTest,
+    testing::Values( std::make_tuple( 1, 0.0, 2.0, true ), std::make_tuple( 1, 0.0, 2.01, false ),
```

Comment:
Add comment here as to whether you are testing for full inclusion of interface pairs, or whether they fall outside the binning proximity. It will help a user/developer if the set of inputs for each test has a comment as to what that test is looking for.

Reply:
Added a comment describing what the test parameters are.